src/NzbDrone.Core/Music/AlbumRepository.cs

```diff
@@ -328,13 +327,6 @@ public void SetMonitored(IEnumerable<int> ids, bool monitored)
             mapper.ExecuteNonQuery(sql);
         }
 
-        public Album FindByName(string cleanTitle)
-        {
-            cleanTitle = cleanTitle.ToLowerInvariant();
-
-            return Query.Where(s => s.CleanTitle == cleanTitle).SingleOrDefault();
-        }
-
```

Comment:
Are we going to need anything like this when we go to album-centric model? I suppose we can always add something better then.

Reply:
We will but it needs more careful handling to deal with special album titles, so probably should be re-done based on the updated `FindByTitle`